lib/MusicBrainz/Server/Edit.pm

```diff
@@ -170,6 +170,7 @@ sub editor_may_edit {
 #  * votes - number of votes to consider the edit as unanimously accepted/rejected
 #  * expire_action - what do do with expired edits without votes
 #  * auto_edit - whether the edit can be automatically accepted for an autoeditor
+#  * always_auto_edit - whether the edit always applies automatically
```

Comment:
For the same reason as above, I don't really like "always" in the comment here. Shouldn't it be "whether the edit applies automatically by default for normal editors"?